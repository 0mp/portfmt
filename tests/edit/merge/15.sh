cat <<EOD | ${PORTEDIT} merge 15.in | diff -L 15.expected -L 15.actual -u 15.expected -
GH_TAGNAME=	ababab
EOD
