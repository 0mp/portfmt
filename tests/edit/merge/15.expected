USES=		autoreconf libtool pkgconfig

USE_LDCONFIG=	yes

USE_GITHUB=	yes
GH_ACCOUNT=	foo
GH_TAGNAME=	ababab
